canonical-web-and-design/maas-ui
# refactor(ui): audit React app CSS - remove unused and use Vanilla icons

## Done

- Audited React app CSS:
  - Removed unused styles and Vanilla overrides that have been fixed.
  - Updated icons to use Vanilla where possible. Follow up will be to remove the aliases.
  - Made utility naming more consistent.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Click around the app - check that everything looks alright.
- Check that the following icons still work as before (though some will have a slightly different look):
  - `p-icon--disconnected`
  - `p-icon--edit`
  - `p-icon--locked`
  - `p-icon--pending`
  - `p-icon--power-error`
  - `p-icon--power-off`
  - `p-icon--power-on`
  - `p-icon--running`
  - `p-icon--tick`
  - `p-icon--timed-out`

## Fixes

Fixes canonical-web-and-design/app-tribe#892
